### ISSUE TITLE
Keyboard focus indicator color contrast is less than 3:1:A11y_VisualStudioCode_Non text Contrast

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode#307581]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier." 

"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".

## GitHub Tags: 
#A11yTCS;#Visual Studio Code Client; #E2E_VisualStudioCodeClient_Win32_Dec2025;#DesktopApp;#FTP;#Win32;#A11ySev2;#A11yMAS;#WCAG1.4.11; #NonText-Contrast;

## Environmental Details: 
App Name: Visual Studio Code Client 
Microsoft Windows 11 Enterprise 
OS Build:  26200.7392

## Repro Steps: 

1. Open Visual Studio Code Insiders editor
2. Install the Microsoft Sentinel extension
3. TAB till Microsoft Sentinel in left navigation and press ENTER key.
4. TAB till Sign in to view lake tables and press ENTER key.
5. Visual studio code popup will open TAB to 'Allow' button and press ENTER key.
6. TAB to popup list item 
7. Observe the issue focus indicator color contrast is less than 3:1.

## Actual Results: 
When navigating through list items using the keyboard, the visual focus indicator displayed on the list item has a color contrast ratio of 1.277:1 against the adjacent background, which is less than 3:1.

## Expected Results: 
Keyboard focus indicator color contrast should be equal or greater than 3:1.

## User Impact: 
Keyboard-only users may lose track of focus while navigating list items.
Low-vision users may not be able to detect the focused item due to insufficient contrast.
This creates a significant usability barrier and increases the risk of unintended actions.

### Attachment:

<img width="1279" height="728" alt="Image" src="https://github.com/user-attachments/assets/446c520f-d823-48bc-8755-f10725421f7b"/>

**Comment by hawkticehurst** [comment]
@kapilvaishna can you confirm what VS Code theme are you using?

**Comment by kapilvaishna** [comment]
@hawkticehurst Dark visual studio

<img width="368" height="41" alt="Image" src="https://github.com/user-attachments/assets/0cdbb2f5-6105-414a-8a68-1a570b810fa3" />

**Comment by hawkticehurst** [comment]
@kapilvaishna thanks for the info! This a very old theme (that comes from Visual Studio) and has, for many years, not been a default theme in VS Code and is a theme that we no longer actively support. I'd recommend looking at the 2026 Dark and 2026 Light themes as the latest generation of default themes we ship and could use accessibility review and support on.

That said, I've checked the 2026 themes and they also do not meet contrast requirements for this scenario so I'll go ahead and work on a PR to fix this!

**Comment by kapilvaishna** [comment]
Issue is not repro in dark theme so closing the bug.

<img width="1280" height="768" alt="Image" src="https://github.com/user-attachments/assets/c497639f-3a76-4a0b-876d-339edee5eacc" />
